libs/orders/src/lib/order-hooks/use-order-validation.tsx

```diff
@@ -64,41 +160,62 @@ export const useOrderValidation = ({
           OrderTimeInForce.GFN,
         ].includes(orderTimeInForce)
       ) {
-        return t(
-          'Only GTT, GTC and GFA are permitted when market is in auction'
-        );
+        return {
+          isDisabled: true,
+          message: t(
+            'Only GTT, GTC and GFA are permitted when market is in auction'
+          ),
+        };
       }
     }
 
     if (fieldErrors?.size?.type === 'required') {
-      return t('An amount needs to be provided');
+      return {
+        isDisabled: true,
+        message: t('An amount needs to be provided'),
+      };
     }
 
     if (fieldErrors?.size?.type === 'min') {
-      return t(`The amount cannot be lower than "${step}"`);
+      return {
+        isDisabled: true,
+        message: t(`The amount cannot be lower than "${step}"`),
+      };
     }
 
     if (fieldErrors?.price?.type === 'required') {
-      return t('A price needs to be provided');
+      return {
+        isDisabled: true,
+        message: t('A price needs to be provided'),
+      };
     }
 
     if (fieldErrors?.price?.type === 'min') {
-      return t(`The price cannot be negative`);
+      return {
+        isDisabled: true,
+        message: t(`The price cannot be negative`),
+      };
     }
 
     if (
       fieldErrors?.size?.type === 'validate' &&
       fieldErrors?.size?.message === ERROR_SIZE_DECIMAL
     ) {
       if (market.positionDecimalPlaces === 0) {
-        return t('No decimal amounts allowed for this order');
+        return {
+          isDisabled: true,
+          message: t('No decimal amounts allowed for this order'),
```

Comment:
updated to the suggested wording

Reply:
@candida-d I think 'fractional' might be a bit confusing.
Would something like 'Order sizes must be whole numbers for this market' make more sense?
Think it's usually best to tell users what they need to do rather than what they can't do where possible, and might avoid confusing terms in this instance.

Reply:
@PeteWilliams Yeah that makes sense, I agree. I’ll suggest a change.

Reply:
@PeteWilliams Yeah that makes sense, I agree. I’ll suggest a change.